sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/Select.java

```diff
@@ -28,8 +28,8 @@
 import org.apache.beam.sdk.schemas.FieldAccessDescriptor;
 import org.apache.beam.sdk.schemas.Schema;
 import org.apache.beam.sdk.schemas.utils.RowSelector;
-import org.apache.beam.sdk.schemas.utils.SelectByteBuddyHelpers;
```

Comment:
I was wondering if SelectByteBuddyHelpers is still used after this removal (and would be able to be deleted). But I don't find it (org.apache.beam.sdk.schemas.utils.SelectByteBuddyHelpers) in the current master.

Reply:
This PR introduces SelectByteBuddyHelpers.

Reply:
I think you are looking at an individual commit.

Reply:
oeps my bad. You are right (early morning). Resoving, looking further.